libraries/ms-helper/src/x86/ads1015.c

```diff
@@ -116,3 +118,152 @@ StatusCode ads1015_read_converted(Ads1015Storage *storage, Ads1015Channel channe
   *reading = (raw_reading * ADS1015_CURRENT_FSR) / ADS1015_NUMBER_OF_CODES;
   return STATUS_CODE_OK;
 }
+#else
+#include <stdlib.h>
+#include <string.h>
+
+#include "ads1015.pb-c.h"
+#include "ads1015_def.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "store.h"
+#include "stores.pb-c.h"
+
+#define ADS1015_CHANNEL_UPDATE_PERIOD_US ADS1015_CONVERSION_TIME_US_1600_SPS
+
+static MxAds1015Store s_store = MX_ADS1015_STORE__INIT;
```

Comment:
Here too the static store prevents us from using more than one ADS1015 in a project, unless I'm misunderstanding something. Somehow the store should be per-storage.